contracts/lbp/LBPManagerFactoryNoAccessControl.sol

```diff
@@ -0,0 +1,158 @@
+/*
+██████╗░██████╗░██╗███╗░░░███╗███████╗██████╗░░█████╗░░█████╗░
+██╔══██╗██╔══██╗██║████╗░████║██╔════╝██╔══██╗██╔══██╗██╔══██╗
+██████╔╝██████╔╝██║██╔████╔██║█████╗░░██║░░██║███████║██║░░██║
+██╔═══╝░██╔══██╗██║██║╚██╔╝██║██╔══╝░░██║░░██║██╔══██║██║░░██║
+██║░░░░░██║░░██║██║██║░╚═╝░██║███████╗██████╔╝██║░░██║╚█████╔╝
+╚═╝░░░░░╚═╝░░╚═╝╚═╝╚═╝░░░░░╚═╝╚══════╝╚═════╝░╚═╝░░╚═╝░╚════╝░
+*/
+
+// SPDX-License-Identifier: GPL-3.0-or-later
+// LBPManager Factory contract. Governance to create new LBPManager contracts.
+// Copyright (C) 2021 PrimeDao
+
+// solium-disable linebreak-style
+pragma solidity 0.8.17;
+
+import "../utils/CloneFactory.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./LBPManager.sol";
+
+/**
+ * @title LBPManager Factory
+ * @dev   Governance to create new LBPManager contract without the onlyOwner modifer for the
+ *        function deployLBPManager(). By removing the access control, everyone can deploy a
+ *        LBPManager from this contract. This is a temporarly solution in response to the
+ *        flaky Celo Safe.
+ */
+contract LBPManagerFactoryNoAccessControl is CloneFactory, Ownable {
+    address public masterCopy;
```

Comment:
wdyt about adding version to LBP contracts?

Reply:
Added it